charts/victoria-logs-single/Chart.yaml

```diff
@@ -25,9 +25,11 @@ annotations:
       url: https://victoriametrics.github.io/helm-charts/
     - name: Docs
       url: https://docs.victoriametrics.com/VictoriaLogs/
-
 dependencies:
-- name: fluent-bit
-  version: "0.46.7"
-  repository: "https://fluent.github.io/helm-charts"
-  condition: fluent-bit.enabled
+  - name: fluent-bit
+    version: "0.46.7"
+    repository: "https://fluent.github.io/helm-charts"
+    condition: fluent-bit.enabled
+  - name: victoria-metrics-common
+    version: "0.0.*"
+    repository: file://../victoria-metrics-common
```

Comment:
Reference to local file system will not work for external users without cloning a repo.
It is needed to publish `common` chart first and then add a reference with proper `repository` + `version` here.